cwms/timeseries/timeseries_txt.py

```diff
@@ -113,6 +115,17 @@ def retrieve_text_ts_json(
 
         return queryCDA(self, end_point, params, headers)
 
+    def retrieve_large_clob(self, url: str) -> str:
+        """
+        Retrieves large clob data greater than 64kb from CWMS data api
+        :param url: str
+            Url used in query by CDA
+        :return: str
+            Large text data
+        """
+        response = requests.get(url)
+        return response.content.decode("utf-8")
```

Comment:
I think the encoding should be parameterized. OK to default to utf-8 since that's the python default

Reply:
done